RoktDemo/UI/Layout Demo/LayoutDemoViewModel.swift

```diff
@@ -54,7 +54,8 @@ class LayoutDemoViewModel: ObservableObject {
         attributes["isDemo"] = "true"
         attributes["layoutId"] = preview.previewId
         attributes["versionId"] = preview.versionId
-        attributes["creativeId"] = preview.creativeIds.joined(separator: ",")
+        attributes["rokt.language"] = preview.roktLanguage
+        attributes["demoConfig"] = preview.demoConfig
```

Comment:
Can you give an example of this demoConfig string?
We try to limit the contents in this model, because the QR code scanner would be slow detecting the code if the amount of data is large.

Reply:
Hi @thomson-t , updated the PR description with a sample QR code content

Reply:
Can we restructure the demoConfig JSON structure and optimise it so that the QR code scanner can perform better?
* We can remove `targetElementSelector`. We don't use that.
* Repetitive `layoutVariantId` and `creativeId` strings can be avoided and we could use a map or tuple.

Reply:
targetElementSelector can be removed and there wound't be any impact.
However, the demoConfig slots is defined on API side and already implemented like that, it is the same for Web and Mobile, hard to change that though. This is the final demoConfig that needs to be send to experiences demo api

Reply:
We can transform it after it is scanned on the mobile side and send whatever format we want. Can we show the QR code with just the information what we need?

Reply:
I agree with the @thomson-t . I already received some complains about the Barcode reader that is not working in some monitors. And now, the speed of testing is reduced because barcode reader is slower that usual for the content that is not required.

Reply:
Do we have other alternative qrcode scanners for iOS to perform better?  Does the size of the content cause the poor performance or there is other factors?

Reply:
The content size caused the poor performance. So we prefer to reduce the content size to what we had a month ago if possible.

Reply:
What would be the suggested json format?
What about the below structure?
```
{
  "tagId": "3157272025983146467",
  "previewId": "3339067146982785095",
  "versionId": "1724121953217",
  "language": "en", // -> rokt.language
  // in the formats of [layoutVariantId, creativeId]
  "layoutVariants" : [["3339067275831803979","3297172498411225129"],["3339067275831803980","3297172884958281775"],["3339067275831803979", "3297172786173902936"],["3339067275831803980", "3297173061051940916"]]
}
```

regarding Barcode reader that is not working in some monitors, is it possible that it is because the QR code is too small that does not work well in low resolution screens? We could potentially increase the size of the QR code.

Reply:
The second one looks good to me.
This will make the format backward compatible, so that if someone want to test the old schema with an old version of the demo app, it still works.

Reply:
Agree with @thomson-t . Second one looks good to me.